media-source-respec.html

```diff
@@ -2195,9 +2279,8 @@ <h2><dfn data-export="">Byte Stream Formats</dfn></h2>
               <p class="note">For example, if the first [=initialization segment=] has 2 audio tracks and 1 video track, then all [=initialization segments=] that follow it in the byte stream MUST describe 2 audio tracks and 1 video track.</p>
             </li>
             <li>[=Track IDs=] are not the same across [=initialization segments=], for segments describing multiple tracks of a single type (e.g., 2 audio tracks).</li>
-            <li>
-              <p>Codecs changes across [=initialization segments=].</p>
-              <p class="note">For example, a byte stream that starts with an [=initialization segment=] that specifies a single AAC track and later contains an [=initialization segment=] that specifies a single AMR-WB track is not allowed. Support for multiple codecs is handled with multiple <a>SourceBuffer</a> objects.</p>
+              <p>Unsupported codec changes occur across across [=initialization segments=].</p>
```

Comment:
HTML validation correctly caught that this <p> (and the next <p> which becomes a <div> in the generated html) both are invalid as direct children of <ol> node. I'll put these back within an <li> like they should be (and like they were before this).